PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,22 @@
 <!--
 Thanks for contributing!
 
-- Add tests! This is what we do afterall. 😉
+- Don't forget to add tests! This is what we do after all. 😉
 - If you haven't already, complete the CLA.
 
 Read more about contributing: https://github.com/cypress-io/cypress/blob/develop/CONTRIBUTING.md
 -->
 
-- close <!--Add issue number here -->
+- Closes <!-- Add issue number here, for example "Closes #1234" -->
+
+# Pre-merge Tasks
+
+<!-- 
+The following tasks must be completed before a PR can be merged.
+You can delete any of these tasks if they are not applicable to the changes in this PR. 
+-->
+
```

Comment:
I would add - "Have tests been written for the bug fix/feature" - I added it as a comment previously, but if we're doing a checklist - this should definitely be part of it.

Reply:
good idea, i also removed the "sign CLA" thing since cla-bot will notify them if it's applicable anyways